src/libYARP_dev/include/yarp/dev/all.h

```diff
@@ -16,6 +16,10 @@
 #include <yarp/dev/CartesianControl.h>
 #include <yarp/dev/ControlBoardInterfaces.h>
 #include <yarp/dev/ControlBoardInterfacesImpl.h>
+#include <yarp/dev/IAxisInfo.h>
```

Comment:
Not 100% sure, but I think they are not necessary, are not enough the `%include` directives you added ?

Reply:
`%include` specify which classes SWIG needs to wrap, while the `#include` in https://github.com/nunoguedelha/yarp/blob/df0a871e9f3e66ce4c15d971f35056589de38604/bindings/yarp.i#L257 actually specify which files are included in the generated C++ files. The alternative is to add this includes in  https://github.com/nunoguedelha/yarp/blob/df0a871e9f3e66ce4c15d971f35056589de38604/bindings/yarp.i#L257 , but I do not think it is a great, unless the `all.h` inclusion is actually removed. 

I agree that `all.h` may be bad, but either we deprecate/remove it officially, or we keep it updated. Do not update it so it is not actually containing all the headers does not seems a good strategy to me.